extensions/azurecore/src/account-provider/auths/azureDeviceCode.ts

```diff
@@ -57,6 +57,11 @@ export class AzureDeviceCode extends AzureAuth {
 
 	}
 
+	public async promptForConsent(resourceId: string): Promise<undefined> {
```

Comment:
it isn't immediately clear to me when this will run.  Does this block any device code scenarios we'd like to support?  Or is this to fix the Conditional Access issue you mentioned on chat?

Reply:
If when getting the token from AAD, the response is "interaction required", because device code is an interaction-less authentication method, they can't use it.

So this will only impact device code auth when the user is using conditional access.